automatic/displayfusion/displayfusion.nuspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>displayfusion</id>
+    <version>0.0</version>
+    <title>DisplayFusion</title>
+    <authors>Binary Fortress Software</authors>
+    <projectUrl>http://www.displayfusion.com</projectUrl>
+    <licenseUrl>https://www.displayfusion.com/License/</licenseUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/d5deb06ccda7af181948eddc25eb4f15ce6c9a6c/icons/DisplayFusion.png</iconUrl>
+    <docsUrl>https://www.displayfusion.com/Help/</docsUrl>
+    <mailingListUrl>https://www.displayfusion.com/Discussions/</mailingListUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    ## Features
```

Comment:
There should also be a few lines of description of the package before this Features section.